spec/features/user_edit_spec.rb

```diff
@@ -0,0 +1,21 @@
+# require 'rails_helper'
+#
+# feature 'edit profile',%{
+#    As a user
+#    I want to edit my profile
+#    So that I can stand out on this fancy site
+# } do
+#
+#   # scenario 'user edits profile page' do
+#   #   visit '/users/edit'
+#   #   click_button 'Update'
+#   #
+#   #   expect(page).to have_content("Current password can't be blank")
+#   # end
+#     #
+#     # scenario 'user edits profile page' do
+#     #   visit '/users/edit'
+#     #
+#     #   expect(page).to have_content("Edit User")
+#     # end
+# end
```

Comment:
Are you using this?  If you're waiting on something else to happen first, leave a note.  Otherwise, eliminate it.